### PR TITLE
Update golang to 1.12.9

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by duderino — APPROVED** [review]

File: prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.1.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ istio_rel_pipeline_spec: &istio_rel_pipeline_spec
     testing: build-pool
 
 istio_rel_pipeline_container: &istio_rel_pipeline_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```
Comment (sdake) [inline prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.1.yaml:13]: i'm not sure how wise it is to change the 1.1 toolchain.

File: prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.2.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ istio_rel_pipeline_spec: &istio_rel_pipeline_spec
     testing: build-pool
 
 istio_rel_pipeline_container: &istio_rel_pipeline_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```
Comment (sdake) [inline prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.2.yaml:13]: or the 1.2 toolchain.

File: prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.3.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ istio_rel_pipeline_spec: &istio_rel_pipeline_spec
     testing: build-pool
 
 istio_rel_pipeline_container: &istio_rel_pipeline_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```
Comment (sdake) [inline prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.3.yaml:13]: 1.3 seems fine - what toolchain was it using prior - 1.12.5?

File: prow/cluster/jobs/istio/istio/istio.istio.release-1.1.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ job_template: &job_template
   path_alias: istio.io/istio
 
 istio_container: &istio_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```
Comment (sdake) [inline prow/cluster/jobs/istio/istio/istio.istio.release-1.1.yaml:8]: 1.1 toolchain change?

File: prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml [hunk 5]
```diff
@@ -5,7 +5,7 @@ job_template: &job_template
   path_alias: istio.io/istio
 
 istio_container: &istio_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```
Comment (sdake) [inline prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml:8]: 1.2 toolchain change?

File: prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.1.yaml (hunk 1 above)
Comment (howardjohn) [reply prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.1.yaml:13]: This is intentional. We have already upgraded 1.1 to go 1.12.5, this just bumps from 1.12.9 which should be safe (patch release). This is recommended by 1.1 release manager @duderino

**Review by sdake — APPROVED** [review]

**Pull request merged**